docker/vm_startup_script.sh

```diff
@@ -1,3 +0,0 @@
-#! /bin/bash
-# Delete old catalystcoop/pudl-etl images
-docker rmi -f $(docker images -q catalystcoop/pudl-etl)
```

Comment:
Why did you remove this? I added this so that different image tags don't build up on the VM.

Reply:
Oh, I removed it because I didn't see it getting run anywhere and figured it was a stale leftover from when we had a local setup.  Does this script have a special magical meaning somewhere else?  If so I can add it back with a comment in the file that explains what it does.

Reply:
Yes! It is magical. It gets run when the VM starts. It feels a little wacky but when I was setting this up originally the disk was filling up with stale images.

@jdangerx any chance this could have played a part in the image not getting updated properly?

Reply:
I don't think so, since this wasn't removed in the branches we were working on. I think it's just Dockerhub <> GCP slowness.

Reply:
Okay! I'll restore it and add a note in the comments inside the file so it's clear it's necessary even though it's not called anywhere in our codebase.